docs/_content/ontwikkelaars/algemeen/zgw-api-landschap-grenzen.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "ZGW API-landschap grenzen"
+date: '07-06-2019'
+weight: 50
+---
+
+In het Common Ground API-landschap worden straks alle bronnen netjes ontsloten 
+middels een API. Er is een API die een gemeentelijk bomenregister ontsluit, een
+API voor Zaken, Documenten, Personen, Medewerkers, etc.
+
+Op dit moment zijn wordt er hard gewerkt aan een set API's die bij elkaar de
+kern vormen voor Zaakgericht werken, de ZGW API's. Deze bestaan op het moment
+van schrijven uit de Zaken API, Documenten API, Zaaktypen API en Besluiten API.
+Deze maken op hun beurt weer gebruik van de Autorisaties API en de Notificaties 
+API.
+
+## Probleemomschrijving
+
+Aangezien dit de eerste set API's is die beschikbaar komt als VNG standaard, is
+het API-landschap nog erg beperkt. Een `Zaak` kent bijvoorbeeld typisch een of 
+meer `Betrokkenen` zoals een `Natuurlijk Persoon` of `Vestiging`. Ook gaat een
+`Zaak` meestal over een of meer `ZaakObjecten` zoals een `Buurt`, 
+`Kunstwerkdeel`, `Ligplaats` of `Wijk`. Allemaal objecten.
+
+Deze objecten achter de Zaak-relaties `Betrokkenen` en `Zaakobjecten` worden 
+nog niet ontsloten middels een API. Er kan dus niet een URL opgegeven worden 
+die verwijst naar zo'n object, zoals hieronder is weergegeven:
+
```

Comment:
Suggestie voor iets andere formulering:

Een `Zaak` kent  typisch een of meer `Betrokkenen` zoals een `Natuurlijk Persoon` of `Vestiging`. Ook gaat een
`Zaak` meestal over een of meer `ZaakObjecten` zoals een `Buurt`, `Kunstwerkdeel`, `Ligplaats` of `Wijk`. Allemaal objecten. Deze objecten 'leven' in eigen registraties die middels API's benaderd moeten worden.
Aangezien de ZGW api's de eerste set API's is die beschikbaar komt als VNG standaard, is
het API-landschap nog erg beperkt.
De objecten achter de Zaak-relaties `Betrokkenen` en `Zaakobjecten` kunnen daarom niet worden ontsloten middels een API. Er kan dus niet een URL opgegeven worden 
die verwijst naar zo'n object, zoals hieronder is weergegeven: